hawkular-command-gateway/hawkular-command-gateway-war/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -24,6 +24,7 @@
       <module name="org.hawkular.bus" />
       <module name="org.hawkular.commons.cassandra-driver" />
       <module name="org.hawkular.commons.hawkular-inventory-paths"/>
+      <module name="deployment.hawkular-inventory-dist.war" services="import"/>
```

Comment:
I definitely think we do not want to pull this in. This is the full inventory war I think. Not really what we want.